packages/frontend/amp/components/elements/ImageBlockComponent.tsx

```diff
@@ -16,17 +17,21 @@ const captionStyle = css`
     line-height: 16px;
     color: ${palette.neutral[46]};
 `;
-const getFallback: (images: Image[]) => Image = images =>
-    images.find(_ => _.fields.isMaster === 'true') || images[0];
 
 export const ImageBlockComponent: React.SFC<{
     element: ImageBlockElement;
     pillar: Pillar;
 }> = ({ element, pillar }) => {
-    const image = getFallback(element.media.allImages);
+    const containerWidth = 600;
+    const image = getImage(element.media.allImages, containerWidth);
```

Comment:
As this is an AMP exclusive function, we can probably leave 600 hard coded in it.

Reply:
I agree this is very minor, but I still quite like naming the argument here.